build.gradle.kts

```diff
@@ -238,15 +235,15 @@ tasks.withType<RunIdeTask> { maxHeapSize = "4G" }
 
 tasks.withType<RunPluginVerifierTask> {
   val maybeEap =
-      if (intellijVersions["eapOfLatestSupportedMajor"] != null)
+      if (IntellijVersions.eapOfLatestSupportedMajor != null)
           listOf(
-              (intellijVersions["eapOfLatestSupportedMajor"] as String).replace(
+              (IntellijVersions.eapOfLatestSupportedMajor!!).replace(
                   "-EAP-(CANDIDATE-)?SNAPSHOT".toRegex(), ""))
       else emptyList()
   ideVersions.set(
       listOf(
-          *(intellijVersions["latestMinorsOfOldSupportedMajors"] as List<String>).toTypedArray(),
-          intellijVersions["latestStable"] as String,
+          *(IntellijVersions.latestMinorsOfOldSupportedMajors).toTypedArray(),
```

Comment:
Nit: hmm maybe join the lists/elements with some `+` operator or whatever is available in Kotlin? I suppose a cast to an array isn't the most elegant solution here